src/tests/Common/testgrouping.proj

```diff
@@ -180,42 +180,6 @@
       <XUnitWrapperDll>$(TestBinDir)JIT\HardwareIntrinsics\JIT.HardwareIntrinsics.XUnitWrapper.dll</XUnitWrapperDll>
     </TestGrouping>
 
-    <TestGrouping Include="$(TestBinDir)JIT\Methodical\a*\**;
```

Comment:
We should undo the changes in this file. This change is only for when we actually merge the JIT/Methodical work.

Reply:
Thanks for pointing that out, reverted in the latest commit.